packages/core/src/+internal/http/contentType.util.ts

```diff
@@ -42,3 +42,6 @@ export const getContentType = (headers: HttpHeaders): string => {
     ? Array.isArray(contentType) ? contentType[0] : String(contentType)
     : '';
 };
+
+export const isJsonContentType = (headerValue: string) =>
+  headerValue.includes('json');
```

Comment:
Maybe stupid question: Are header values guaranteed to be case-insensitive? If not, should we lowercase the value before checking?

Reply:
I'm not sure if `APPLICATION/JSON` is a correct Content-Type value 🤔

Reply:
/shrug
Guess people can open bug reports, if so :)

Reply:
yup 👍